CMakeSettings.json

```diff
@@ -67,14 +67,14 @@
     },
     {
       "name": "x64-Debug-tests",
-      "generator": "Ninja",
+      "generator": "Visual Studio 15 2017 Win64",
```

Comment:
I don't see how these changes are related to the PR.

Reply:
Roped in changes made by VVDN from PR #1107 . This section is for WSL built enclave tests running under Visual Studio 2017 and not exercised by Windows CI.